mmv1/third_party/terraform/website/docs/r/cloudfunctions_function.html.markdown

```diff
@@ -146,7 +146,13 @@ Eg. `"nodejs10"`, `"nodejs12"`, `"nodejs14"`, `"python37"`, `"python38"`, `"pyth
 * `source_archive_object` - (Optional) The source archive object (file) in archive bucket.
 
 * `source_repository` - (Optional) Represents parameters related to source repository where a function is hosted.
-  Cannot be set alongside `source_archive_bucket` or `source_archive_object`. Structure is [documented below](#nested_source_repository).
+  Cannot be set alongside `source_archive_bucket` or `source_archive_object`. Structure is [documented below](#nested_source_repository). It must match the pattern `projects/{project}/locations/{location}/repositories/{repository}`.* 
+
+* `docker_repository` - (Optional, Beta) User managed repository created in Artifact Registry optionally with a customer managed encryption key. If specified, deployments will use Artifact Registry. This is the repository to which the function docker image will be pushed after it is built by Cloud Build.
```

Comment:
That comment is slightly outdated now. Previously, the plan was to fully migrate from the GCF to AR. Due to anticipated disruptions for the customers, instead, we are now releasing AR as an optional feature. That is, the GCF won't use set it by default, except one specific case: when AR is explicitly specified via `docker_registry` field (so-called Google managed AR), but that capability is still in beta. In other words, the default AR repository will only be created when docker_registry is explicitly set to Artifact Registry type, but a particular repo is not specified. Overall, it doesn't seem to fall under the [definition](https://www.terraform.io/plugin/sdkv2/schemas/schema-behaviors#computed) of Computed, at least until we introduce `docker_registry` support and even then it will apply only to one specific scenario, in which it will still remain "user configurable".

Reply:
Added a brief note on what to expect when this parameter is not specified.

Reply:
Just to double-check - did this change get pushed?

Reply:
Sorry, the documentation update specifically actually didn't. I have pushed it along with the latest doc change.